local-cli/generator-ios/templates/app/AppDelegate.m

```diff
@@ -47,6 +47,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
                                                       moduleName:@"<%= name %>"
                                                initialProperties:nil
                                                    launchOptions:launchOptions];
+  rootView.backgroundColor = [[UIColor alloc] initWithRed:1.0f green:1.0f blue:1.0f alpha:1];
```

Comment:
I see your point. In that case, you could use `+[UIColor colorWithRed:green:blue:alpha:]`, which avoids the need to call `+[UIColor alloc]`.

Also, why use `1.0f` for every float value besides alpha, which is simply `1`?

All of these are nits of course, feel free to ignore. :)